### ISSUE TITLE
Add Accessible Format Provider Type

### ISSUE THREAD

**Issue by currycl** [CEDStandards/CEDS-Elements#802]
This is for capturing needs not currently supported by the CEDS model.  Please do not send or share actual data as examples in this issue or in attachments. 

**Author(s)**
Cynthia Curry

**Authoring Organization(s)**

**Email address**
aem@cast.org

**Use Case Title**
Add Accessible Format Provider Type

**Use Case Description** 
The source from which accessible formats are acquired. 

**Use Case Background**
The AEM Center workgroup, funded by OSEP, built out 5 CEDS connections focusing on students with disabilities receiving Accessible Formats.  Throughout the in person and virtual meetings with 5 states, the addition of Accessible Format Provider Type was identified as a new element. 

**Location of Element in the Domain Entity Schema**
Learning Resources> Learning Resource

**Comment by currycl** [comment]
Adding formal proposal
[Uploading Accessible Format Provider Type.docx…]()

**Comment by aujaleemoore** [comment]
I think that we need to be clear about what options are available and ensuring that they are clear, and not duplicative. For example, prison braille networks could be added. However, many states procure the braille from their IMC/IRC (Instructional Resource Center). I think, where possible, this list should be exhaustive. However, reducing duplication will provide clearer data and be more user friendly.

**Comment by currycl** [comment]
Adding proposal
[Accessible Format Provider Type.docx](https://github.com/CEDStandards/CEDS-Elements/files/13369025/Accessible.Format.Provider.Type.docx)

**Comment by MNilsen1** [comment]
Are we only considering AMPs for the format provider type? Where would other audio files (audible, SORA, etc.) fit in (or not)?

**Comment by brianwojcik** [comment]
I am wondering if, in addition to common accessible media producers, there are options that may be available like 'Third-Party Commercial Service (e.g., Audible)' or ' Public Domain Resources (e.g., Project Gutenberg)' to allow for options beyond typica AMPs.

**Comment by twinton65** [comment]
Some of these are actual names of specific providers (e.g., APH, NLS) while others are generic types of providers. Seems a little strange to call all of them 'Provider Type'. Maybe instead call the category 'Provider Name/Type'.

**Comment by jackie-hughes** [comment]
Updated Accessible Format Provider Type to include NPBN (National Prison Braille Network)
[Accessible.Format.Provider.Type - Updated.docx](https://github.com/CEDStandards/CEDS-Elements/files/13531048/Accessible.Format.Provider.Type.-.Updated.docx)

**Comment by jackie-hughes** [comment]
The above proposal is a solution for this use case. As a community, please review. If no objections exist for the proposed solution, it will be approved as part of the CEDS standard 30 days following the announcement of this proposal as outlined in the OSC Use Case Rubrics/Process (The OSC Use Case Rubrics/Process can be found here: https://github.com/CEDStandards/CEDS-Elements/tree/master/doc).

**Comment by ccurry9825** [comment]
> Are we only considering AMPs for the format provider type? Where would other audio files (audible, SORA, etc.) fit in (or not)?

For the purpose of providing accessible formats, the sources of those accessible formats must be accessible media producers (AMPs). By definition, an accessible format is a material that has been converted to be usable by an eligible person under Section 121/Chafee. That conversion is completed by an AMP. Audible and Sora provide audio books for use by everyone; that is, their materials are not provided under Section 121/Chafee for the sole use of eligible persons.

**Comment by ccurry9825** [comment]
> Some of these are actual names of specific providers (e.g., APH, NLS) while others are generic types of providers. Seems a little strange to call all of them 'Provider Type'. Maybe instead call the category 'Provider Name/Type'.

This is a good point. In light of other comments, perhaps a further clarification could be achieved by naming it "Accessible Format Provider Type/Name"

**Comment by jackie-hughes** [comment]
In response to feedback, we are modifying the proposal, specifically the option set to include SEA, LEA, School, and Non-profit organization. This adjustment, combined with the addition of Accessible Format Provider to Organization Type (Issue 813), enables the utilization of Organization Type, Organization Name, and Organization Identifier. This allows the collection of non-profit organization names while assigning an Organization Identifier to the corresponding Accessible Format Provider Type.

**Comment by ccurry9825** [comment]
I agree with the rationale for the modification to the proposal. Based on two considerations, I recommend a minor adjustment. First, "SEA" does not accurately represent sources of accessible formats. Second, the option set does is not representative of the National Library Service, its network of state libraries, nor IMCs/IRCs. Given these two considerations, I recommend that SEA be replaced with "National or State Service."

**Comment by jackie-hughes** [comment]
[AccessibleFormatProviderType-.Updated Options.docx](https://github.com/CEDStandards/CEDS-Elements/files/14112151/AccessibleFormatProviderType-.Updated.Options.docx)
The attached document is the proposed solution for this use case. As a community, please review. If no objections exist for the proposed solution, it will be approved as part of the CEDS standard 60 days following the announcement of this proposal as outlined in the OSC Use Case Rubrics/Process (The OSC Use Case Rubrics/Process can be found here: https://github.com/CEDStandards/CEDS-Elements/tree/master/doc).

**Comment by ccurry9825** [comment]
After continued reflection, the community might consider further refining the option description for "National or State Service Organization." For clarification and to be as specific as possible, the description could be "State- or Federally-funded Service Organization." This would help prevent confusion with commercial purveyors, such as Audible, Amazon, etc., that provide materials in audio, closed captions, audio description, etc. on the market for anyone to purchase. However, these are not accessible format providers, which are exclusive to eligible persons who require a material to be converted or enhanced because of a disability. Thanks for your patience on this iterative process.

**Comment by manndoris-CEPI** [comment]
Well guess I needed to read a bit farther before commenting on #813!  Are we moving toward identifying IF an Accessibility provider is used AND adding "Accessibility Provider" to Organization Type and identifying the Provider in the Organization Identifier?? Do we really need this level of detail? How will this data be used? I don't see a use for this level of detail in Michigan. We just identify if a student requires assistance or modifications for assessments. We expect the districts to maintain the details, however, so that may be where the additional details would be useful for local vendors.

**Comment by jackie-hughes** [comment]
Proposed Solution for Issue 802

[AccessibleFormatProviderType-.Updated.Options.docx](https://github.com/CEDStandards/CEDS-Elements/files/14635967/AccessibleFormatProviderType-.Updated.Options.docx)

The attached document is the proposed solution for this use case. As a community, please review. If no objections exist for the proposed solution, it will be approved as part of the CEDS standard 60 days following the announcement of this proposal as outlined in the OSC Use Case Rubrics/Process (The OSC Use Case Rubrics/Process can be found here: https://github.com/CEDStandards/CEDS-Elements/tree/master/doc)
